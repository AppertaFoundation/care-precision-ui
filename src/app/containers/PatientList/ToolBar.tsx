import React from 'react';
import { BoxWrapper } from 'components';
import { Box } from '@material-ui/core';

interface Props {
  children: React.ReactNode[];
}
export const ToolBar: React.FC<Props> = ({ children }) => {
  return (
    <BoxWrapper>
      <Box display="flex" flexWrap="nowrap" alignItems="center" p={1}>
        <Box width="80%">{children[0]}</Box>
        <Box width="20%">{children[1]}</Box>
      </Box>
    </BoxWrapper>
  );
};
