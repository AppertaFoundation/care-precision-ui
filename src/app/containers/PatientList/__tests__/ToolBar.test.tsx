import * as React from 'react';
import { render } from '@testing-library/react';
import { ToolBar } from '../ToolBar';

const renderToolBar = (props: Parameters<typeof ToolBar>[number]) =>
  render(<ToolBar {...props} />);

describe('<ToolBar />', () => {
  it('should match the snapshot', () => {
    const ToolBar = renderToolBar({
      children: [<p>Hello</p>, <p>Word</p>],
    });
    expect(ToolBar.container.firstChild).toMatchSnapshot();
  });
});
