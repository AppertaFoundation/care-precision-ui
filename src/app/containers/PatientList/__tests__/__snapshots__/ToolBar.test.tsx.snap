// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ToolBar /> should match the snapshot 1`] = `
<div
  class="makeStyles-root-1"
>
  <div
    class="MuiBox-root MuiBox-root-2"
  >
    <div
      class="MuiBox-root MuiBox-root-3"
    >
      <p>
        Hello
      </p>
    </div>
    <div
      class="MuiBox-root MuiBox-root-4"
    >
      <p>
        Word
      </p>
    </div>
  </div>
</div>
`;
